PowerShell/Tools/BitTitanManagement.ps1

```diff
@@ -707,12 +707,12 @@ function GoogleHelper-GetDomainUsers([string]$domainName)
 
 function GoogleHelper-SetMailboxForward([string]$emailAddress, [string]$targetAddress)
 {
-    $forward = Set-MigrationWizGoogleEmailForward -Ticket (MWHelper-GetTicket) -Environment (MWHelper-ChooseEnvironment) -EmailAddress $emailAddress -Enable $true -ForwardTo $targetAddress -Action ([BitTitan.Powershell.Core.GoogleEmailForwardAction]::Archive)
+    $forward = Set-MigrationWizGoogleEmailForward -Ticket (MWHelper-GetTicket) -Environment (MWHelper-ChooseEnvironment) -EmailAddress $emailAddress -Enable $true -ForwardTo $targetAddress -Action Archive -Verbose
```

Comment:
Did you intend to keep the "-Verbose" or was it just for debugging purposes?

Reply:
Yeah it is on purpose, coz most information are written as verbose. Without this, customer won't see any result, which is confusing.